core/src/tile/tileManager.cpp

```diff
@@ -212,18 +233,20 @@ void TileManager::updateTileSet(TileSet& _tileSet, const ViewState& _view,
                     enqueueTask(_tileSet, visTileId, _view);
                 }
             } else {
-                if (entry.isLoading()) {
+
+                if (entry.isLoading() && entry.rastersPending() == 0) {
                     if (newTiles) {
                         // check again for proxies
                         updateProxyTiles(_tileSet, visTileId, entry);
                     }
                     m_tilesInProgress++;
```

Comment:
```entry.rastersPending() == 0``` should not be necessary -- if a proxy tile became available for use it does not matter if the actual tile has pending rasters